libs/mep/ace1068/section-metadata/sticky-section.js

```diff
@@ -0,0 +1,98 @@
+import { createTag } from '../../../utils/utils.js';
+import { getMetadata, getDelayTime } from './section-metadata.js';
+import { getGnavHeight } from '../../../blocks/global-navigation/utilities/utilities.js';
+
+function handleTopHeight(section) {
+  const topHeight = getGnavHeight();
+  section.style.top = `${topHeight}px`;
+}
+
+function promoIntersectObserve(el, stickySectionEl, options = {}) {
+  return new IntersectionObserver((entries, observer) => {
+    entries.forEach((entry) => {
+      if (el.classList.contains('close-sticky-section')) {
+        window.removeEventListener('resize', handleTopHeight);
+        observer.unobserve(entry.target);
+        return;
+      }
+
+      const { target } = entry;
+      const { isIntersecting } = entry;
```

Comment:
This can be one line
```suggestion
      const { target, isIntersecting  } = entry;
```